telegram/ext/_application.py

```diff
@@ -1316,6 +1316,8 @@ async def process_error(
             job (:class:`telegram.ext.Job`, optional): The job that caused the error.
 
                 .. versionadded:: 14.0
+            coroutine (:term:`coroutine function`): The coroutine that caused the error. Only
+                present if the coroutine was run with :attr:`block=False <Handler.block>`.
```

Comment:
* the argument is optional.
* "only present" doesn't really make sense for an argument. this should instead go into the docs of `CallbackContext.coroutine`. suggestion for the text: `Only present in error handlers if the error was caused by a coroutine run with App.create_task or a handler callback with block=False` (the fact that block=false uses app.create_task doesn't have to be documented here IMO)